### PR TITLE
Revised State Median Income (SMI) Ceilings 2024 for all states

### LINKED ISSUE #4808: California Revised State Median Income (SMI) Ceilings
[Reference](https://cdss.ca.gov/Portals/9/Additional-Resources/Letters-and-Notices/CCBs/2024/CCB_24-12.pdf?ver=2024-07-02-114041-930)

### LINKED ISSUE #4829: Revised State Median Income (SMI) Ceilings 2024 for all states
Will cover Issue #4808 
Ref: https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHWAP_Att1SMITable_FY2023.pdf

### PR DESCRIPTION
**Pull request by vrathi101**
Fixes #4829
Fixes #4808

### CODE REVIEW

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Could you add uprating while you're here?

**Comment by vrathi101** [comment]
> Could you add uprating while you're here?

there seem to be uprating issues with gov.hhs.uprating I think because it doesn't like that the parameter is defined for October?

**Comment by MaxGhenis** [comment]
I think we want to index to some earnings measure instead, like Social Security

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/hhs/smi/amount.yaml [hunk 1]
```diff
@@ -164,4 +216,7 @@ metadata:
       href: https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHEAP_IM03%20Attachment1%20SMITable_FY2022.pdf
     - title: State Median Income (SMI) by Household Size for Optional Use in FFY 2022 and Mandatory Use in LIHEAP for FFY 2023
       href: https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHEAP_Att1SMITable_FY2023.pdf
+    - title: State Median Income (SMI) by Household Size for Mandatory Use in LIHWAP for FY 2024
+      href: https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHWAP_Att1SMITable_FY2023.pdf
   period: year
+  uprating: gov.ssa.uprating
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/hhs/smi/amount.yaml:222]:
```suggestion
  uprating: calibration.gov.cbo.income_by_source.adjusted_gross_income
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Realized AGI would better reflect SMI as it's a broader income measure

Could you adjust the tests if needed?

**Comment by MaxGhenis** [comment]
Re-running to see if the main Ubuntu test failure is intermittent (it's not the normal documentation one)

**Comment by vrathi101** [comment]
> Re-running to see if the main Ubuntu test failure is intermittent (it's not the normal documentation one)

does it look good?

**Comment by MaxGhenis** [comment]
No, it's still failing in the test suite rather than documentation. The job getting killed could be due to resource limitations or a timeout, though it shouldn't be a timeout since GitHub Actions limits to 360 minutes and this failed in 33. @nikhilwoodruff or @anth-volk any ideas?

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -54,4 +54,4 @@ jobs:
         run: make
       - name: Test documentation builds
         if: matrix.os == 'ubuntu-latest'
-        run: make documentation
+        run: make documentation
```
Comment (PavelMakarchuk) [inline .github/workflows/pr.yaml:57]: @vrathi101 was this supposed to be changed?

Comment (vrathi101) [reply .github/workflows/pr.yaml:57]: > @vrathi101 was this supposed to be changed?

no, I was trying to revert it but I don't know how cuz there's no extra empty line in my version of the file

Comment (MaxGhenis) [reply .github/workflows/pr.yaml:57]: we should have an extra empty line

**Pull request merged**